app/kumactl/cmd/get/get_dataplanes.go

```diff
@@ -28,11 +29,11 @@ func newGetDataplanesCmd(pctx *listContext) *cobra.Command {
 			}
 
 			dataplanes := mesh.DataplaneResourceList{}
```

Comment:
Is it possible to write all this `newGet${POLICY}Cmd` in a generic manner? Instead of `mesh.DataplaneResourceList{}` I think we can use `registry.Global().NewList(type)`?